### DISCUSSION TITLE
Think of minio as disk io, I'm not sure if what I described is accurate.

### DISCUSSION THREAD

**Discussion by Mrmanbanpai** [minio/minio#19815] [Q&A] [answered]
It's like this. I received a request to use minio as a disk driver so that the application's behavior when accessing the disk remains unchanged, but the actual access process becomes api access to minio. I don't know if you have any relevant open source projects that I can use. .
The reason why I ask this question is because I saw this article posted by the community https://blog.min.io/filesystem-on-object-store-is-a-bad-idea/. I think this has something to do with me. The thing is the same, but I need proof to prove it.

Looking forward to your reply, thank you

**Comment by klauspost** [comment, marked as answer]
Your application should be rewritten to use S3 directly.

**Reply by Mrmanbanpai** [reply]
I get what you meant now, thanks.
